tests/test__mongomock.py

```diff
@@ -2489,6 +2507,141 @@ def test__aggregate_filter(self):
             'cond': {'$lt': ['$$this.price', 100]},
         }}}}])
 
+    def test__aggregate34(self):
+        pipeline = [
+            {'$project': {
+                'c': {'$abs': [-2]},
+                'd': {'$floor': 2.3},
+                'e': {'$ln': None},
+                'f': {'$exp': '$non_existent_key'},
+                'g': {'$divide': [7, 3]},
+                'h': {'$log': [None, 1]},
+                'i': {'$mod': [1, None]},
+                'j': {'$pow': [None, None]},
+                'k': {'$subtract': ['$non_existent_key', 1]},
+                'l': {'$sum': []},
+                'm': {'$multiply': []},
+                'n': {'$add': '$a'},
+                'o': {'$multiply': [4]},
+                'p': {'$add': [1, 2, 3]},
+                'q': {'$multiply': [1, '$non_existent_key']},
+                'r': {'$add': '$non_existent_key'},
+                's': {'$multiply': [1, None]},
+                't': {'$add': [None, 1]},
+                'u': {'$multiply': ['$a', '$b', 4]},
+                'v': {'$multiply': '$b'}
+            }}
+        ]
+        self.cmp.compare.aggregate(pipeline)
+
+    def test__aggregate_exception(self):
```

Comment:
Please add a TODO (for me) to do those comparisons more easily with a single command.

Reply:
Fixed.